caesars-palace/blocks/header/header.js

```diff
@@ -110,18 +204,25 @@ export default async function decorate(block) {
       if (section) section.classList.add(`nav-${c}`);
     });
 
+    // Remove the text in the link
+    nav.querySelector('.nav-brand a').innerHTML = '';
+
     const navSections = nav.querySelector('.nav-sections');
     if (navSections) {
-      navSections.querySelectorAll(':scope > ul > li').forEach((navSection) => {
-        if (navSection.querySelector('ul')) navSection.classList.add('nav-drop');
-        navSection.addEventListener('click', () => {
-          if (isDesktop.matches) {
-            const expanded = navSection.getAttribute('aria-expanded') === 'true';
-            toggleAllNavSections(navSections);
-            navSection.setAttribute('aria-expanded', expanded ? 'false' : 'true');
-          }
-        });
+      const newDiv = document.createElement('div');
+      newDiv.classList.add('local-nav');
+      const localNavTitle = document.createElement('div');
+      localNavTitle.classList.add('local-nav-title');
+      localNavTitle.setAttribute('aria-expanded', true);
+      localNavTitle.innerHTML = 'Property Links';
+      newDiv.appendChild(localNavTitle);
+      while (navSections.hasChildNodes()) newDiv.appendChild(navSections.firstChild);
```

Comment:
I don't know about eslint here, but my personal preference is to have brackets around a while's logic, so it's easier to read and see in case it foes not do what's intended

Reply:
comes down to personal preference. Since it is a one liner, it makes sense to consolidate. If it had more than one line, then brackets would be necessary.